docs/content/operations/api-reference.md

```diff
@@ -230,6 +230,20 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+##### PUT
```

Comment:
PUT is supposed to be idempotent, so I think this should be POST instead: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT